src/index.html

```diff
@@ -10,7 +10,38 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form
+    action="#"
+    method="post"
+    data-qa="big"
+    class="big"
+    >
```

Comment:
```suggestion
    <form
      action="#"
      method="post"
      data-qa="big"
      class="big"
    >
```
pay attention to code style - [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 5

fix it evrywhere